applications/facemask/libs/detectors/x86/util.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
```

Comment:
I can not find where this function is used.

Reply:
o0o0ps.